client/src/App.js

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 
 import Header from "./components/Header";
+import Timer from "./components/Timer";
 import "./App.css";
 
 function App() {
     return (
         <div className="App">
             <Header />
+            <Timer />
```

Comment:
To make sure that we don't have conflicts with PRs, it is easier if there are no changes in the App.js file. @Chuppay will handle rendering the component in issue #81